StylaSEO/Controllers/Frontend/Stylaapi.php

```diff
@@ -264,10 +264,14 @@ public function getEKPrices($detail) {
             $productEKPseudoPrice = $price['pseudoPrice'];
         }
 
-        return [
-            "price" => $productEKPrice,
-            "pseudoPrice" => $productEKPseudoPrice,
-        ];
+        // factor might be 0 or 1 which means its the base price
+        $factor = $currencyInfo['factor'];
+        $factor = $factor == 0 ? 1 : $factor;

```

Comment:
not sure if I get this part: doesn't this mean that you are always forcing factor to be 1?